Week-1/cattyboard.html

```diff
@@ -0,0 +1,256 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Pesto > Assignment Week 1</title>
+</head>
+<style>
```

Comment:
Avoid inline styles rather create separate files.( Read about- Separation of concerns/Single responsibility)